src/main/anovos/data_report/basic_report_generation.py

```diff
@@ -101,6 +102,7 @@ def anovos_basic_report(
     output_path=".",
     run_type="local",
     print_impact=True,
+    mlflow_config: dict = None,
```

Comment:
For backwards compatibility, please use `Dict` (`from typing import Dict`) and if possible specify the types completely:

```python
from typing import Union, Dict, Any

...

mlflow_config: Union[Dict[str, Any], None] = None
```
```